web/source/keyboards/kmwkeyboards.ts

```diff
@@ -1055,13 +1101,12 @@ namespace com.keyman.keyboards {
           badName = x['keyboardid'].substr(0,1).toUpperCase()+x['keyboardid'].substr(1);
         }
 
-        this.serverUnavailable(badName+' keyboard not found.');
-        return;
+        return new Error(badName+' keyboard not found.');
```

Comment:
We should make this a constant like all the others.

Reply:
The `badName` portion's not constant, though.  Otherwise, I'd agree with you.

Guess I should check if there's any `printf`-like string functions available and make it insertable into a format string?

Reply:
You could have done `return new Error(KEYBOARD_NOT_FOUND + badName);` I guess where `const KEYBOARD_NOT_FOUND = 'Keyboard not found: ';`